app/src/main/java/com/yeonae/chamelezone/view/login/JoinFragment.kt

```diff
@@ -94,7 +132,7 @@ class JoinFragment : Fragment(), JoinContract.View {
                 Toast.LENGTH_SHORT
             ).show()
             else -> {
-                if (!email_layout.isErrorEnabled && !password_layout.isErrorEnabled && !nickname_layout.isErrorEnabled) {
+                if (!email_layout.isErrorEnabled && !password_layout.isErrorEnabled && !nickname_layout.isErrorEnabled && checkedEmail && checkedNickname) {
```

Comment:
```suggestion
                if (!email_layout.isErrorEnabled && 
                !password_layout.isErrorEnabled && 
                !nickname_layout.isErrorEnabled && 
                checkedEmail && 
                checkedNickname) {
```
이런 줄바꿈에 대한 convention도 있어요

Reply:
수정했습니다